com.unity.netcode.gameobjects/Components/Interpolator/BufferedLinearInterpolator.cs

```diff
@@ -186,7 +186,14 @@ public T Update(float deltaTime, double renderTime, double serverTime)
 
                     if (t < 0.0f)
                     {
-                        throw new OverflowException($"t = {t} but must be >= 0. range {range}, RenderTime {renderTime}, Start time {m_StartTimeConsumed}, end time {m_EndTimeConsumed}");
+                        // There is no mechanism to guarantee renderTime to not be before m_StartTimeConsumed
```

Comment:
>  it will still cause the unsightly, confidence-shaking errors almost every time people run our flagship Boss Room demo

Can you back this affirmation up? From the analysis Noel and I did, this would only kick in when:

```            
            if (m_NbItemsReceivedThisFrame > k_BufferCountLimit)
            {
                if (m_LastBufferedItemReceived.TimeSent < sentTime)
```
Basically when a whole lot of stuff is received but it's all old stuff. e.g. you paused the client or there was some other massive message flow issue. There doesn't seem to be much to do at this point. 

If we want really bullet-proof interpolation, the alternative I see is a re-write but with cleaner up-front requirements (e.g. I don't blame the code, but I'm realistic about how we got here). Such a rewrite seems a bit out-of-scope.

Reply:
I agree an overhaul is out of scope, per our conversation, let's leave this in and prove out that it doesn't show in BR.  If it doesn't then leaving the error message in is appropriate as a way to actually show that something has been broken in the implementation.